com.github.suzie97.communique.yml

```diff
@@ -0,0 +1,77 @@
+pe# This is the same ID that you've used in meson.build and other files
+app-id: com.github.suzie97.communique
+
+# Instead of manually specifying a long list of build and runtime dependencies,
+# we can use a convenient pre-made runtime and SDK. For this example, we'll be
+# using the runtime and SDK provided by elementary.
+runtime: io.elementary.Platform
+runtime-version: '6'
+sdk: io.elementary.Sdk
+
+# This should match the exec line in your .desktop file and usually is the same
+# as your app ID
+command: com.github.suzie97.communique
+
+# Here we can specify the kinds of permissions our app needs to run. Since we're
+# not using hardware like webcams, making sound, or reading external files, we
+# only need permission to draw our app on screen using either X11 or Wayland.
+finish-args:
+  - '--socket=pulseaudio'
+  - '--socket=fallback-x11'
+  - '--share=ipc'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--share=network'
+  - '--metadata=X-DConf=migrate-path=/com/github/suzie97/communique/'
+  - '--talk-name=org.gnome.OnlineAccounts'
+  - '--own-name=com.github.suzie97.communique.ArticleView'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--system-talk-name=org.freedesktop.Accounts'
+
+# This section is where you list all the source code required to build your app.
+# If we had external dependencies that weren't included in our SDK, we would list
+# them here.
+modules:
+  - name: libgee
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.4.tar.xz
+        sha256: 524c1bf390f9cdda4fbd9a47b269980dc64ab5280f0801b53bc69d782c72de0e
+
+  - name: libpeas
+    buildsystem: meson
+    sources:
+    - type: git
+      url: https://gitlab.gnome.org/GNOME/libpeas.git
+      branch: master
+
+  - name: rest
+    cleanup:
+      - '/include'
+      - '*.la'
+      - '/lib/pkgconfig'
+      - '/share'
+    sources:
+    - type: archive
+      url: "https://download.gnome.org/sources/rest/0.8/rest-0.8.1.tar.xz"
+      sha256: 0513aad38e5d3cedd4ae3c551634e3be1b9baaa79775e53b2dba9456f15b01c9
+
+  - name: gnome-online-accounts
+    sources:
+    - type: archive
+      url: https://download-fallback.gnome.org/sources/gnome-online-accounts/3.40/gnome-online-accounts-3.40.0.tar.xz
+      sha256: 585c4f979f6f543b77bfdb4fb01eb18ba25c2aec5b7866c676d929616fb2c3fa
+
+  - name: gumbo
+    sources:
+    - type: archive
+      url: https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz
+      sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+
+  - name: communique
+    buildsystem: meson
+    config-opts:
+    - '--buildtype=release'
+    sources:
+      - type: dir
+        path: .
```

Comment:
this won't work. there is no source here. This should be either a git repository or an archive.